src/current/v23.1/expire-past-backups.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Expire Past Backups
+summary: Guidance to move or delete past backups in cloud storage. 
+toc: true
+docs_area: manage
+---
+
+This page describes how to expire past backups from cloud storage. As an operational best practice, we recommend [taking regular backups](take-full-and-incremental-backups.html) of your data and using cloud storage for your backups.
+
+The process for expiring past backups from cloud storage will vary depending on the specific cloud storage service used. For guidance on connecting to different storage options or using other authentication parameters, read [Use Cloud Storage](use-cloud-storage.html).
+
+## Step 1. Determine your retention policy
+
+Before you can expire past backups, first determine your backup retention policy. With incremental backups in the /incrementals directory, you can apply different lifecycle/retention policies from cloud storage providers to the /incrementals directory as needed.
+
+For example: Retain [full backups](take-full-and-incremental-backups.html#full-backups) for 30 days and retain [incremental backups](take-full-and-incremental-backups.html#incremental-backups) for 7 days. 
+
+## Step 2. Identify past backups
+
+Once you have determined your retention policy, identify the past backups that need to be expired. You can use the creation time of the backup files to identify backups that are older than your desired retention period.
+
+Since individual files within a backup have different creation times and will as a result be moved or deleted at different times. Configuring the policy that moves or deletes them to run a day after your intended retention period ensures all files within a retained backup remain available for the entire desired retention period.
```

Comment:
> Since individual files within a backup have different creation times and will as a result be moved or deleted at different times. 
Since individual files within a backup have different creation times, they will be moved or deleted at different times.


>  Configuring the policy that moves or deletes them to run a day after your intended retention period ensures all files within a retained backup remain available for the entire desired retention period.
Change to active voice.
To ensure that all files within a retained backup remain available during the entire deisred retention period, configure the policy that moves or deletes them to run one day AFTER your intended retention period.